.circleci/config.yml

```diff
@@ -895,6 +916,14 @@ jobs:
       - provision-gke-cluster:
           cluster-id: rhel
 
+  provision-slim-cluster:
```

Comment:
```suggestion
  provision-rhel-slim-cluster:
```

Reply:
how come we want to add "rhel" here? I don't think it's needed since the other provision steps don't specify rhel in the name (just in the cluster-id)

Reply:
Makes sense.